graphene_django/views.py

```diff
@@ -54,6 +55,20 @@ def instantiate_middleware(middlewares):
         yield middleware
 
 
+class ExceptionLoggingExecutionContext(ExecutionContext):
+
+    """An execution context which logs exceptions."""
+
+    def handle_field_error(
+        self,
+        error: GraphQLError,
+        return_type,
+    ) -> None:
+
+        print_tb(error.original_error.__traceback__)
```

Comment:
Could we have that using a logger instead? so that it can be enabled/disabled? Also could we document how to do that

Reply:
I'm not a really good fan of using a logger here because it's a traceback that is on multiples lines by definition. What do you think about adding a settings here: https://github.com/loft-orbital/graphene-django/blob/loft-main/graphene_django/settings.py#L24 that control whether or not we're using by default this `ExceptionLoggingExecutionContext` ?